src/server/sams/api/produce/assets.py

```diff
@@ -119,6 +119,32 @@ def unlock_asset(asset_id: str):
     return service.patch(ObjectId(asset_id), updates)
 
 
+@assets_produce_bp.route('/produce/assets/user/<user_id>', methods=['PATCH'])
+def unlock_asset_by_user(user_id: str):
+    """
+    Uses user_id and unlocks the corresponding assets
+    """
+
+    service = get_asset_service()
+    query = {
+        'lock_user': user_id
+    }
+    assets = service.get(req=None, lookup=query)
```

Comment:
It would also be better to use elasticsearch rather than mongodb
i.e.
```
service.search({
    'query': {
        'bool': {
            'must': [
                {'term': {'lock_user': user_id}},
                {'term': {'lock_session': session_id}}
            ]
        }
    }
}
```